releasenotes/notes/add-aws_service-region-resourcename-tags-for-aws-sdk-requests-186ea33075ef3168.yaml

```
@@ -0,0 +1,6 @@
+---
+other:
+  - |
+    Adds aws_service, region, <resource_name> such as queue_name tags to Boto, Pynamodb, Aiobotocore, and Botocore intercepted requests such as SNS Publish or SQS Send Message.
+    Motivation for this is Serverless inferred related resources, pivoting from traces to aws integration metrics out of the box and much more outlined
+    in google doc `Consistent APM span tags for AWS requests`
```

We can call out the change in the feature section and include update instructions.

```suggestion
feature:
  - |
    Makes span tag names consistent with metrics and logs for AWS integrations.
upgrade:
  - |
    AWS span tag names used in queries for trace retention filters, span metrics, monitors, dashboards and notebooks must be updated: 
    * For ``aws.region``, use ``region``.
    * For ``aws.sqs.queue_name``, use ``queuename``.
    * For ``aws.dynamodb.table_name``, use ``dynamodb``.
    * For ``aws.kinesis.stream_name``, use ``streamname``.
    * For ``aws.redshift.cluster_identifier``, use ``clusteridentifier``.
    * For ``aws.s3.bucket_name``, use ``bucketname``.
```